examples/tcp_server/src/main.rs

```diff
@@ -161,6 +161,9 @@ fn handle_connection_event(
                     break;
                 }
                 Ok(n) => {
+                    // echo back what we received and ignore any errors while doing so
+                    let _ = connection.write_all(&received_data[bytes_read..(bytes_read + n)]);
```

Comment:
This is just an example so it hardly matters, but I'm curious why we're ignoring errors here rather than using `?` like in the previous version.

Reply:
If the other does a shutdown on the receiving side, we still want all what it sent and display that.